src/CC-BY-4.0.xml

```diff
@@ -35,7 +35,7 @@
          the licensed material may still be restricted for other reasons, including because others have
          copyright or other rights in the material. A licensor may make special requests, such as asking that
          all changes be marked or described. Although not required by our licenses, you are encouraged to
-         respect those requests where reasonable. More considerations for the public.</p>
+         respect those requests where reasonable. More considerations for the public<alt match=": wiki.creativecommons.org/Considerations_for_licensees|\.">: wiki.creativecommons.org/Considerations_for_licensees</alt></p>
```

Comment:
Upstream has an unfortunate typo underscore [here][1] (since [2014-01-10][2]), so I think we want something like:

```xml
… More<alt match=" |_"> </alt>considerations…
```

as well.

[1]: https://creativecommons.org/licenses/by/4.0/legalcode.txt
[2]: https://web.archive.org/web/20140110003133/https://creativecommons.org/licenses/by/4.0/legalcode.txt

Reply:
Pointed this out to them at https://github.com/creativecommons/creativecommons.org/pull/714 ... maybe they'll fix going forward.

I'll go ahead and add your suggestion in any case.